src/constants.ts

```diff
@@ -71,3 +71,12 @@ export type QuickPicksCache = { cache: QuickPickItem[], next: string | null };
 
 // Originally from the Docker extension: https://github.com/microsoft/vscode-docker/blob/main/src/constants.ts
 export const DOCKERFILE_GLOB_PATTERN = '**/{*.[dD][oO][cC][kK][eE][rR][fF][iI][lL][eE],[dD][oO][cC][kK][eE][rR][fF][iI][lL][eE],[dD][oO][cC][kK][eE][rR][fF][iI][lL][eE].*}';
+
+export const unsavedChangesTrueContextValue: string = 'unsavedChanges:true';
+export const unsavedChangesFalseContextValue: string = 'unsavedChanges:false';
+
+export const revisionDraftTrueContextValue: string = 'revisionDraft:true';
```

Comment:
This isn't imported by anything. Happening in a follow-up PR?

Reply:
You are correct, I probably could comment this out or take it out of this PR.

Reply:
It definitely feels weird having revisionDraft and unsavedChanges, but there is an extremely small nuanced difference that ends up being useful for showing context commands